cutscene_engine/data/test_stroheim_dialogue1.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Do we need all these test dialogues committed to the repo? Maybe we can keep one or two along with some code snippets and label them as examples.

Reply:
test_stroheim_dialogue1.json and test_stroheim_dialogue2.json are all that we need yes. It was just an oversight not deleting the others considering that I just simply forgot they were there. It kind of sucks that Godot doesn't support .json to be read in-engine and has to be open via a manual window.